auth-web/src/components/auth/manage-business/Certify.vue

```diff
@@ -40,6 +42,10 @@ export default class Certify extends Vue {
   // local variable
   protected isCertified = false
 
+  get currentUserName (): string {
+    return this.currentUser.lastName + ', ' + this.currentUser.firstName
```

Comment:
this can be written 
return `${this.currentUser.lastName}, ${this.currentUser.firstName}`

Reply:
Okay. I'll fix it.